app-misc/vcontrold/Manifest

```diff
@@ -0,0 +1 @@
+DIST vcontrold-0.98.4.tar.gz 84351 SHA256 e73fbc69189fc76a1eae1698ea0ea155ae1a5ed8591af22cc26e0bbef8ac64f5 SHA512 41e553cdd81a6e0567844e78de8a8f7cfdb42990c9739a1c6e92db6f146d7b2d23620cc8fc32be92c645f09bfac18bcc27b2c216a248a04eda9b64b1439ec397 WHIRLPOOL 0ed059aab550563ecd3d11f593477c96eaedb606e44623686a6e8164ec85a783c8727f9fec26ca2a31ef8a1eb3eb22ffe928031f0f6fd51719ef262b57e2ca5b
```

Comment:
This is now an invalid Manifest. We should only see BLAKE2B and SHA512. Rebase your fork or update metadata/layout.conf to match the hash declaration in gentoo/gentoo.

Reply:
Done.